src/main/resources/view/MainWindow.fxml

```diff
@@ -8,7 +8,7 @@
 <?import javafx.scene.layout.*?>
 <?import javafx.stage.*?>
 
-<fx:root minHeight="800.0" minWidth="800.0" onCloseRequest="#handleExit" title="Dictionote" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root maximized="true" minHeight="300.0" minWidth="500.0" onCloseRequest="#handleExit" title="Dictionote" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Does the `maximized="true"` part starts the program in maximized window mode by default?

Reply:
yap